src/Symfony/Installer/DemoCommand.php

```diff
@@ -145,11 +145,13 @@ private function displayInstallationResult()
             ));
         }
 
+        $serverRunCommand = version_compare($this->version, '2.6.0', '>=') && extension_loaded('pcntl') ? 'server:start' : 'server:run';
```

Comment:
You cannot select a version of the demo application, can you? And the one that is going to be installed is using Symfony 3 already, isn't it?